hypothesis-python/docs/conf.py

```diff
@@ -259,6 +259,8 @@ def setup(app):
 .. |BackgroundWriteDatabase| replace:: :class:`~hypothesis.database.BackgroundWriteDatabase`
 .. |RedisExampleDatabase| replace:: :class:`~hypothesis.extra.redis.RedisExampleDatabase`
 
+.. |is_hypothesis_test| replace:: :func:`~hypothesis.is_hypothesis_test`
```

Comment:
I think in this case I'd just inline the target definition here, since we're not referring it that often and it's not that much longer without custom link text.  Does seem to be somewhere where we just have diverging taste though, and I don't mind you going with yours!

Reply:
My thought process is that in the future I don't want to have to think about what sphinx role to use (remember if it's called func or method), forget the tilde, etc. Maybe a skill issue on my end? I'm happy to admit this might be a personal crutch and that we shouldn't have a goal of `|`-references for every possible hypothesis object, though I weakly think it's a good thing.